rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -710,7 +710,7 @@ object Reduce {
                        case (GByteArray(lhs), GByteArray(rhs)) =>
                          for {
                            _ <- costAccountingAlg.charge(
-                                 STRING_APPEND_COST * (lhs.size() + rhs.size())
+                                 PREPEND_COST * (lhs.size() + rhs.size())
```

Comment:
Instead of `PREPEND_COST * (lhs.size() + rhs.size())`
just
`PREPEND_COST * lhs.size()`

Reply:
I will make this change, but it seems wrong to me that the cost of concating two ByteArrays would not match the cost of concating two Strings.

Reply:
In FP languages to concatenating two collections needs to traverse the first one and glue the 2nd one (which is `O(1)`).